### PR TITLE
Update header to use search query in the nav bar

### PR DESCRIPTION
**Pull request by banklesss**
Related to [YOMO-502](https://emurgo.atlassian.net/browse/YOMO-502)

[YOMO-502]: https://emurgo.atlassian.net/browse/YOMO-502?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/TxHistory/TxHistoryNavigator.tsx [hunk 1]
```diff
@@ -91,18 +91,10 @@ export const TxHistoryNavigator = () => {
...
-          {() => (
-            <Boundary>
-              <SelectTokenFromListScreen />
-            </Boundary>
-          )}
-        </Stack.Screen>
+          name="send"
+          component={SelectTokenFromListScreenNavigator}
+          options={{headerShown: false}}
+        />
```
Comment (banklesss) [inline src/TxHistory/TxHistoryNavigator.tsx:97]: Needed to be able to use the custom search header

Comment (banklesss) [reply src/TxHistory/TxHistoryNavigator.tsx:97]: https://github.com/Emurgo/yoroi-mobile/pull/2416/files?diff=split&w=0#diff-fd919e7597a7cd4b1fab1d574a4c614c5af104cf90ae95d69b72294b1ed12c67R1-R65

File: src/TxHistory/TxHistoryNavigator.tsx [hunk 2]
```diff
@@ -171,13 +163,9 @@ const messages = defineMessages({
...
     defaultMessage: '!!!Scan QR code address',
   },
-  selectAssetTitle: {
-    id: 'components.send.selectasset.title',
-    defaultMessage: '!!!Select asset',
-  },
   selectedTokensTitle: {
     id: 'components.send.selectedtokensscreen.title',
-    defaultMessage: '!!!Selected tokens',
+    defaultMessage: '!!!Assets Addeds',
```
Comment (wolverineks) [inline src/TxHistory/TxHistoryNavigator.tsx:168]: what is `"Addeds"`?

File: src/Search/SearchHeader.tsx [hunk 3]
```diff
@@ -16,47 +16,39 @@ export const useSearchHeaderOptions = ({placeHolderText, title}) => {
...
-          right: 0,
-          bottom: 0,
-          top: 0,
-        },
-        headerLeftContainerStyle: styles.disableFlex,
-        headerLeft: () => null,
+        headerBackTitleVisible: false,
       }
 
   return {searchHeaderOptions}
```
Comment (wolverineks) [inline src/Search/SearchHeader.tsx:35]: why is this an object?

**Pull request closed without merging**